platform/tests/TESTS/mbed_functional/callback/main.cpp

```diff
@@ -886,7 +886,7 @@ Case cases[] = {
     Case("Testing callbacks with 2 uint64s", test_dispatch2<uint64_t>),
     Case("Testing callbacks with 3 uint64s", test_dispatch3<uint64_t>),
     Case("Testing callbacks with 4 uint64s", test_dispatch4<uint64_t>),
-#if !defined(__ICCARM__) && (!defined(MBED_ROM_SIZE) || (MBED_ROM_SIZE >= TEST_MIN_REQ_ROM_SIZE))
+#if !defined(__ICCARM__) && (!defined(MBED_CONF_TARGET_ROM_SIZE) || (MBED_CONF_TARGET_ROM_SIZE >= TEST_MIN_REQ_ROM_SIZE))
```

Comment:
Would this change not cause test failures on targets that use the `MBED_ROM_SIZE` macro? We only define `MBED_CONF_TARGET_ROM_SIZE` for `LPC1114` don't we? Maybe I'm missing something. Please explain in the commit message.

Reply:
Thanks for your review comment. I am about to push commit to keep MBED_ROM_SIZE check

Reply:
Maybe we don't want to skip if `MBED_ROM_SIZE` isn't _defined_ (so we can define it for targets that incorrectly don't define it), but this new change means we also don't check if `MBED_ROM_SIZE >= TEST_MIN_REQ_ROM_SIZE`. Isn't that a problem?

Reply:
What ever targets has mbed-rom-size config in targets.json which has more than expected rom size to include this test cases (This is not correct way) but I am working on copying all targets ram size and rom size from index.json into targets.json

Reply:
OK. So, now that we've added this new parameter to all targets that define `mbed-rom-size`, this should work.